ouroboros-consensus-shelley/test/Test/ThreadNet/RealTPraos.hs

```diff
@@ -121,9 +122,14 @@ prop_simple_real_tpraos_convergence k
                   genesisConfig
                   (coreNodes !! fromIntegral nid)
             , rekeying    = Nothing
-            , txGenExtra  = ShelleyTxGenExtra keySpace
+            , txGenExtra  = ShelleyTxGenExtra $ Gen.GenEnv keySpace constants
             }
 
+    constants :: Gen.Constants
+    constants = Gen.defaultConstants
+      { Gen.frequencyMIRCert = 0
```

Comment:
I'm assuming generating them caused failures. Do we eventually want to generate them?

Reply:
Yes. This is hopefully one of the things we'll discuss with @nfrisby later.